specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2021-02-10/service.json

```diff
@@ -16848,26 +18623,6 @@
       },
       "x-ms-discriminator-value": "A2A"
     },
-    "RecoveryPlanA2ADetails": {
-      "description": "Recovery plan A2A specific details.",
-      "type": "object",
-      "allOf": [
-        {
-          "$ref": "#/definitions/RecoveryPlanProviderSpecificDetails"
-        }
-      ],
-      "properties": {
-        "primaryZone": {
-          "description": "The primary zone.",
-          "type": "string"
-        },
-        "recoveryZone": {
-          "description": "The recovery zone.",
-          "type": "string"
-        }
-      },
-      "x-ms-discriminator-value": "A2A"
-    },
     "RecoveryPlanA2AInput": {
```

Comment:
Please get this verified by vinayak once

Reply:
resolving as no issue found